### PR TITLE
update schools page to reflect schools_prekto12 data

### PR DESCRIPTION
**Pull request by gregbunce**

### CODE REVIEW

**Comment by gregbunce** [comment]
@ZachBeck do you want to give this a look before we have steve do his magic?
https://deploy-preview-1482--gis-utah-gov.netlify.app/data/society/schools-libraries/

**Comment by ZachBeck** [comment]
Looks good. Would it be better to add a space in title banner to read SCHOOLS PRE K TO 12 rather than SCHOOLS PREK TO 12? That's my only nit pick.

**Comment by gregbunce** [comment]
cool, thanks!  sure, we could do that.  i'm on it.

**Comment by gregbunce** [comment]
@steveoh when you get a min, it's ready for your touch.  thx!

File: data/society/schools-libraries/index.html [hunk 1]
```diff
@@ -11,17 +11,12 @@
   - schools
 categories: []
 date: 2012-06-13 14:52:45 -0600
-Schools:
-  downloads:
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mN1Bobk5uYmNodW8&export=download
-      label: 'Society.Schools: File Geodatabase'
-    - url: https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mOXB2NWpKaEJkMWc&export=download
-      label: 'Society.Schools: Shapefile'
```
Comment (steveoh) [inline data/society/schools-libraries/index.html:19]: **thought**: Has this been deprecated already?

File: data/society/schools-libraries/index.html [hunk 2]
```diff
@@ -42,20 +37,65 @@
 abstract="This data package contains a statewide schools dataset and a statewide libraries dataset." %}
 <div class="grid package">
   <div class="grid__col grid__col--12-of-12">
-    <h3 id="Schools">Schools</h3>
+    <h3 id="SchoolsPreKto12">Schools Pre K to 12</h3>
```
Comment (steveoh) [inline data/society/schools-libraries/index.html:40]: **suggestion**: remove abbreviations

```suggestion
    <h3 id="SchoolsPreKto12">Schools (Preschool and kindergarten through 12)</h3>
```

File: data/society/schools-libraries/index.html [hunk 3]
```diff
@@ -42,20 +37,65 @@
...
-    <h5>Related Resources</h5>
+    <img class="inline-text-left outline" src="{% link images/Schools.png %}" alt="SchoolsPreKto12" />
+    <p>This dataset contains point locations for preschool and K-12 schools in the state of Utah - updated for the 2019-2020 school year. Higher education and adult vocational training are not included in the dataset.</p>
+    <p>The Schools Pre K to 12 dataset was built in collaboration with <a href="https://wfrc.org/">Wasatch Front Regional Council</a> (WFRC), the <a href="https://schools.utah.gov/">Utah State Board of Education</a> (USBE), and the <a href="https://gis.utah.gov/">Utah Automated Geographic Reference Center</a> (AGRC). The dataset includes public and private K-12 school locations with enrollment and demographics info by grade for public schools (charter and traditional) reported to the USBE. While some preschools are included, this is not a comprehensive statewide dataset for preschools.</p>
```
Comment (steveoh) [inline data/society/schools-libraries/index.html:45]: **suggestion**: use real table name
**nitpick**: always use link tags for internal links. There should never be gis.utah.gov links in a page.

```suggestion
    <p>The <code>Schools_PreKto12</code> dataset was built in collaboration with <a href="https://wfrc.org/">Wasatch Front Regional Council</a> (WFRC), the <a href="https://schools.utah.gov/">Utah State Board of Education</a> (USBE), and the <a href="{% link index.html %}">Utah Automated Geographic Reference Center</a> (AGRC). The dataset includes public and private K-12 school locations with enrollment and demographics info by grade for public schools (charter and traditional) reported to the USBE. While some preschools are included, this is not a comprehensive statewide dataset for preschools.</p>
```

File: data/society/schools-libraries/index.html [hunk 4]
```diff
@@ -42,20 +37,65 @@
...
+    <p>Schools marked as 'closed' have not been included in this data. More information regarding schools in Utah can be found on the <a href="https://schools.utah.gov/">Utah State Board of Education website</a>.</p>
+    <p>The Schools Pre K to 12 will be updated annually in conjunction with USBE’s <a href="https://schools.utah.gov/data/reports">annual report</a> cycle.</p>
```
Comment (steveoh) [inline data/society/schools-libraries/index.html:47]: suggestion: use real table name
```suggestion
    <p>The <code>Schools_PreKto12</code> will be updated annually in conjunction with USBE’s <a href="https://schools.utah.gov/data/reports">annual report</a> cycle.</p>    
```

File: data/society/schools-libraries/index.html (hunk 1 above)
Comment (gregbunce) [reply data/society/schools-libraries/index.html:19]: that's the next step.   but, yeah, that layer will be deprecated.

Comment (steveoh) [reply data/society/schools-libraries/index.html:19]: ok, in the future we'd want to save this for when that issue hits porter but if you are confident that it is being deprecated then I am ok with it.

**Comment by gregbunce** [comment]
@steveoh , good suggestions, thanks!  i think i committed them all (individually, not as a batch - oops).

File: data/society/schools-libraries/index.html (hunk 1 above)
Comment (gregbunce) [reply data/society/schools-libraries/index.html:19]: we'll actually add another layer to this page for higher education schools.  i hope to get that going this week.

**Comment by ZachBeck** [comment]
I reached out to Spencer Jenkins for higher ed locations and he put me in touch with a couple folks that should have the data. Hopefully I'll hear back soon.

**Comment by steveoh** [comment]
I'm not sure that we want to reference the internal table name in the article like I suggested since no one will ever see it. I'm now thinking we should say something more generic like "This dataset ..."

**Pull request merged**